### ISSUE TITLE
NullPointerException on AestheticCoordinatorLayout

### ISSUE THREAD

**Issue by boswelja** [afollestad/aesthetic#20]
- [x] I have verified there are no duplicate active or recent bugs, questions, or requests
- [x] I have verified that I am using the latest version of Aesthetic (see the README)
- [x] I have given my issue description title
- [x] I have read over the documentation (before asking questions on how to do something).

 - Aesthetic version: `0.4.2`
 - Device OS version: `7.1.2+`
 - Device: Any

```gradle
java.lang.NullPointerException: Attempt to invoke virtual method 'void android.support.design.widget.AppBarLayout.removeOnOffsetChangedListener(android.support.design.widget.AppBarLayout$OnOffsetChangedListener)' on a null object reference
                                                                         at com.afollestad.aesthetic.AestheticCoordinatorLayout.onDetachedFromWindow(AestheticCoordinatorLayout.java:181)
                                                                         at android.view.View.dispatchDetachedFromWindow(View.java:17041)
                                                                         at android.view.ViewGroup.dispatchDetachedFromWindow(ViewGroup.java:3649)
                                                                         at android.view.ViewGroup.dispatchDetachedFromWindow(ViewGroup.java:3641)
                                                                         at android.view.ViewGroup.dispatchDetachedFromWindow(ViewGroup.java:3641)
                                                                         at android.view.ViewGroup.dispatchDetachedFromWindow(ViewGroup.java:3641)
                                                                         at android.view.ViewGroup.dispatchDetachedFromWindow(ViewGroup.java:3641)
                                                                         at android.view.ViewGroup.dispatchDetachedFromWindow(ViewGroup.java:3641)
                                                                         at android.view.ViewGroup.dispatchDetachedFromWindow(ViewGroup.java:3641)
                                                                         at android.view.ViewGroup.dispatchDetachedFromWindow(ViewGroup.java:3641)
                                                                         at android.view.ViewRootImpl.dispatchDetachedFromWindow(ViewRootImpl.java:3457)
                                                                         at android.view.ViewRootImpl.doDie(ViewRootImpl.java:6306)
                                                                         at android.view.ViewRootImpl.die(ViewRootImpl.java:6283)
                                                                         at android.view.WindowManagerGlobal.removeViewLocked(WindowManagerGlobal.java:458)
                                                                         at android.view.WindowManagerGlobal.removeView(WindowManagerGlobal.java:396)
                                                                         at android.view.WindowManagerImpl.removeViewImmediate(WindowManagerImpl.java:123)
                                                                         at android.app.ActivityThread.handleDestroyActivity(ActivityThread.java:4467)
                                                                         at android.app.ActivityThread.-wrap5(Unknown Source:0)
                                                                         at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1652)
                                                                         at android.os.Handler.dispatchMessage(Handler.java:105)
                                                                         at android.os.Looper.loop(Looper.java:164)
                                                                         at android.app.ActivityThread.main(ActivityThread.java:6535)
                                                                         at java.lang.reflect.Method.invoke(Native Method)
                                                                         at com.android.internal.os.Zygote$MethodAndArgsCaller.run(Zygote.java:240)
                                                                         at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:767)
```
 
###### Reproduction Steps
Launch app for first time, or dismiss app from recents

###### Expected Result
No errors, successful launch

###### Actual Result
Crashing

**Comment by MasoodFallahpoor** [comment]
Same error here. It happens when I run the following code
```java
Aesthetic.get()
    .activityTheme(R.style.AppThemeLight)
    .isDark(false)
    .colorPrimaryRes(R.color.light_theme_color_primary)
    .colorAccentRes(R.color.colorAccent)
    .colorStatusBarRes(R.color.light_theme_color_primary)
    .textColorPrimaryRes(R.color.light_theme_text_color_primary)
    .textColorSecondaryRes(R.color.light_theme_text_color_secondary)
    .colorWindowBackgroundRes(R.color.light_theme_window_background)
    .apply();
```

**Comment by afollestad** [comment]
Fixed in 0.4.3

https://github.com/afollestad/aesthetic/releases/tag/0.4.3
